src/tests/multihost/alltests/test_sssd_nss.py

```diff
@@ -44,6 +44,14 @@ def test_avoid_interlocking_among_threads(self, multihost,
             2. Should succeed
             3. Should succeed
         """
+        version_fedora = 0
+        version_rel_cent = 0
+        if "Fedora" in multihost.client[0].distro:
+            version_fedora = int(re.search(r'\d+', multihost.client[0].distro).group())
+        else:
+            version_rel_cent = float(re.findall(r"\d+\.\d+", multihost.client[0].distro)[0])
+        if version_rel_cent < 9 and version_fedora < 35:
```

Comment:
This should be `or`

Reply:
no as initial value of version_rel_cent and version_fedora is 0 so this will work

Reply:
think about a condition where value of rhel is 9.2 and test is  running on fedora.
the value of version_rel_cent = 9.2 but value of version_fedora =0 
now if i put "or" it will skip the test , which should not